content/code-of-conduct.md

```diff
@@ -3,3 +3,135 @@ title: "Code of Conduct"
 menu: "main"
 weight: 6
 ---
+
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```

Comment:
```suggestion
reported to the community leaders responsible for enforcement at
conduct@distribits.live.
```

or

```suggestion
reported to the organizers@distribits.live.
```
following the perspective email address in https://github.com/datalad/distribits-2024/pull/14 . I would for now go for the latter.

Reply:
I think it would be good to have a transparent point of contact, not an email address that could be read by potentially many people who are not named. This makes it also easier to approach a point of contact in person. IIRC @jsheunis kindly volunteered to take over this task

Reply:
```suggestion
reported to Stephan Heunis, the community member responsible for enforcement, at
[INSERT CONTACT METHOD].
```

should we use my email address, a dedicated email address, or something else?

Reply:
what would you be most comfortable with?

Reply:
I'd rather it went to some event-specific address, also useful for longevity. IIUC, @mih created the listserv address, but that isn't necessarily useful here?

Reply:
I *think* it would go to a bunch of people or who ever maintains the list. But I looked up email aliases, and the FZJ IT Services allow you to apply for an email alias. So you could have an email alias like "stephan-distribits@fz-juelich.de" and emails send to this address would land in your work in box. Would that work?